src/Utils/ProductWishlist.php

```diff
@@ -16,16 +16,24 @@ class ProductWishlist
 {
     private $wishlistPrefix = 'wishlist_product_';
     private $session;
+    private $flashBag;
 
     public function __construct(SessionInterface $session)
     {
         $this->session = $session;
+        $this->flashBag = $this->session->getBag('flashes');
     }
 
     public function add(Product $product)
     {
-        $id = $product->getId();
-        $this->session->set($this->makeKey($id), $id);
+        if (5 > count($this->getWishlist())) {
```

Comment:
Please move wishlist limit (5) to class constant. Use it here and in flash message